src/aws_lc_rs_algs.rs

```diff
@@ -124,6 +124,39 @@ pub static RSA_PKCS1_2048_8192_SHA512: &dyn SignatureVerificationAlgorithm = &Aw
     verification_alg: &signature::RSA_PKCS1_2048_8192_SHA512,
 };
 
+/// RSA PKCS#1 1.5 signatures using SHA-256 for keys of 2048-8192 bits,
+/// with illegally absent AlgorithmIdentifier parameters.
```

Comment:
Maybe worth some note here-ish that they're only intended to verify signatures on existing certs for compat? Something like the content in the PR desc.

Ditto for the ring_algs.rs version if you agree.